bigtable/bigtable.go

```diff
@@ -453,9 +461,53 @@ func prefixSuccessor(prefix string) string {
 	return string(ans)
 }
 
+// ReadIterationStats captures information about the iteration of rows or cells over the course of
+// a read, e.g. how many results were scanned in a read operation versus the results returned.
+type ReadIterationStats struct {
```

Comment:
Need comments for fields in the `ReadIterationStats` struct.

Reply:
Done.